pkg/api/update.go

```diff
@@ -16,7 +16,7 @@
 
 package api
 
-//nolint
+// nolint
```

Comment:
Please remove this unrelated change. It would also be wrong, BTW. `//nolint` is a maching readable comment-directive so by golang conventions it should not have a leading space.

Reply:
done